### DISCUSSION TITLE
[failed]:采集容器内部日志失败

### DISCUSSION THREAD

**Discussion by xjdszp** [alibaba/ilogtail#238] [Help] [answered]
按照社区给的文档，采集容器内部日志的配置，部署好后，没有看到kafka中采集的数据,配置如下
kind: ConfigMap
apiVersion: v1
metadata:
  name: ilogtail-user-cm
  namespace: ilogtail
data:
  json_log.yaml: |
    enable: true
    inputs:
      - Type: file_log
        LogPath: /root/log
        FilePattern: "json.log"
        DockerFile: true
        DockerIncludeLabel:
          io.kubernetes.container.name: json-log
    processors:
      - Type: processor_json
        SourceKey: content
        KeepSource: false
        ExpandDepth: 1
        ExpandConnector: ""
    flushers:
      - Type: flusher_kafka
        Brokers:
          - kafka-headless.elk.svc.cluster.local:9092
        Topic: logtail

apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: json-log
  name: json-log
  namespace: default
spec:
  replicas: 1
  selector:
    matchLabels:
      app: json-log
  template:
    metadata:
      labels:
        app: json-log
    spec:
      containers:
        - args:
            - >-
              mkdir -p /root/log; while true; do date +'{"time":"+%Y-%m-%d
              %H:%M:%S","message":"Hello, iLogtail!"}' >>/root/log/json.log;
              sleep 10; done
          command:
            - /bin/sh
            - '-c'
            - '--'
          image: 'alpine:3.9.6'
          name: json-log
          volumeMounts:
            - mountPath: /etc/localtime
              name: volume-localtime
      volumes:
        - hostPath:
            path: /etc/localtime
            type: ''
          name: volume-localtime

**Comment by messixukejia** [comment, marked as answer]
可以登入到logtail容器中，按照https://ilogtail.gitbook.io/ilogtail-docs/configuration/logging 做下基本的排查。
